### ISSUE TITLE
Requesting an individual resource - request produced using path parameter instead of ID in brackets encoded in URL

### ISSUE THREAD

**Issue by BrandU** [davidmoten/odata-client#393]
Hey David,

**Quick question - is this somehow achievable using the generated client:**

So desired HTTP request form: 

``` {serviceUri}/odata/Constructions(1296481).... ```

OData generated client by your generator produces: 

``` {serviceUri}/odata/Constructions/1296481/.... ```

For the invocation looking like this:

``` ugkkClient.constructions(id).select("Id").expand(...)... ``` 

It seems that OData standard itself implicitly goes with the 1st form: [link](https://www.odata.org/getting-started/understand-odata-in-6-steps/) (see step 2)

**Am I doing something wrong here?**

Thank you!

**Comment by BrandU** [comment]
Hey David,

Found a way how to configure the PathStyle.

```  HttpService httpService = new ApacheHttpClientHttpService( //
                new Path(ugkkAPIUrl, PathStyle.IDENTIFIERS_IN_ROUND_BRACKETS), this::createClient, (url, m) -> m); ```
                
 B.

**Comment by davidmoten** [comment]
Yep, that looks like it. This is similar (and shows setting of PathStyle):

https://github.com/davidmoten/odata-client#how-to-create-a-client-for-your-odata-service-non-microsoft-api-with-bearer-tokens
